manifests/a/Azul/Zulu/12/JRE/12.1.3/Azul.Zulu.12.JRE.locale.zh-CN.yaml

```
@@ -0,0 +1,33 @@
+# Created with YamlCreate.ps1 v2.4.1 Dumplings Mod $debug=QUSU.CRLF.7-4-2.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.6.0.schema.json
+
+PackageIdentifier: Azul.Zulu.12.JRE
+PackageVersion: 12.1.3

```

```suggestion
PackageVersion: "12.1"
```
Please update file path to match this change.